.github/workflows/deploy_test_branch.yml

```diff
@@ -20,24 +20,17 @@ jobs:
         with:
           node-version-file: '.node-version'
 
+      - name: Install pnpm
+        run: npm i -g pnpm
```

Comment:
Let`s use cache like we do in [dx-toolkit](https://github.com/fingerprintjs/dx-team-toolkit/blob/main/.github/workflows/build-typescript-project.yml#L63)

And also we can use github action `pnpm/action-setup` to install pnpm (but we should do it before installing node)